scala/scala.bzl

```diff
@@ -253,53 +252,60 @@ scala_repl = rule(
     toolchains = ['@io_bazel_rules_scala//scala:toolchain_type'],
 )
 
-def scala_repositories(scala_version = "2.11.11"):
-  major_version_underscore = scala_version[:scala_version.find(".", 2)].replace(
-      ".", "_")
+def scala_repositories(scala_version = _default_scala_version()):
+  major_version_underscore = _extract_major_version_underscore(scala_version)
 
   _new_scala_repository("scala_default", scala_version)
 
   # scalatest has macros, note http_jar is invoking ijar
```

Comment:
@johnynek should I open an issue about changing these http_jar to use `scala_maven_import_external`?